diff --git a/mailnews/imap/src/nsImapFlagAndUidState.cpp b/mailnews/imap/src/nsImapFlagAndUidState.cpp
--- a/mailnews/imap/src/nsImapFlagAndUidState.cpp
+++ b/mailnews/imap/src/nsImapFlagAndUidState.cpp
@@ -171,17 +171,17 @@ NS_IMETHODIMP nsImapFlagAndUidState::Res
 {
   PR_CEnterMonitor(this);
   if (!howManyLeft)
     fNumberOfMessagesAdded = fNumberDeleted = 0; // used space is still here
   if (m_customFlagsHash.IsInitialized())
     m_customFlagsHash.EnumerateRead(FreeCustomFlags, nsnull);
   memset(fFlags, 0, sizeof(imapMessageFlagsType) * fNumberOfMessageSlotsAllocated);
   m_customFlagsHash.Clear();
-  memset(fUids.Elements(), fNumberOfMessageSlotsAllocated * sizeof(nsMsgKey), 0);
+  memset(fUids.Elements(), 0, fNumberOfMessageSlotsAllocated * sizeof(nsMsgKey));
   fPartialUIDFetch = PR_TRUE;
   PR_CExitMonitor(this);
   return NS_OK;
 }
 
 
 // Remove (expunge) a message from our array, since now it is gone for good
