--- icedove-3.0/comm-1.9.1/mail/app/profile/all-thunderbird.js	2009-12-05 02:53:47.000000000 +0100
+++ icedove-3.0.new/comm-1.9.1/mail/app/profile/all-thunderbird.js	2009-12-17 11:11:15.450649343 +0100
@@ -73,12 +73,12 @@
 // App-specific update preferences
 
 // Whether or not app updates are enabled
-pref("app.update.enabled", true);               
+pref("app.update.enabled", false);               
 
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make     
 // the UI easier to construct.
-pref("app.update.auto", true);
+pref("app.update.auto", false);
 
 // Defines how the Application Update Service notifies the user about updates:
 //
@@ -373,7 +373,7 @@
 pref("alerts.totalOpenTime", 3000);
 
 // analyze urls in mail messages for scams
-pref("mail.phishing.detection.enabled", true);
+pref("mail.phishing.detection.enabled", false);
 // If phishing detection is enabled, allow fine grained control
 // of the local, static tests
 pref("mail.phishing.detection.ipaddresses", true); 
