diff -ur comm-1.9.1/mozilla/modules/libpr0n/encoders/png/nsPNGEncoder.cpp comm-1.9.1.new/mozilla/modules/libpr0n/encoders/png/nsPNGEncoder.cpp
--- comm-1.9.1/mozilla/modules/libpr0n/encoders/png/nsPNGEncoder.cpp	2010-01-11 18:42:57.000000000 +0100
+++ comm-1.9.1.new/mozilla/modules/libpr0n/encoders/png/nsPNGEncoder.cpp	2010-02-02 09:45:17.079023061 +0100
@@ -135,7 +135,7 @@
 
   // initialize
   mPNG = png_create_write_struct(PNG_LIBPNG_VER_STRING,
-                                 png_voidp_NULL,
+                                 NULL,
                                  ErrorCallback,
                                  ErrorCallback);
   if (! mPNG)
