--- icedove-3.1.2/mozilla/mozilla/toolkit/xre/nsXREDirProvider.cpp~	2010-08-03 02:28:31.000000000 +0300
+++ icedove-3.1.2/mozilla/mozilla/toolkit/xre/nsXREDirProvider.cpp	2010-08-17 15:46:17.972676534 +0300
@@ -328,7 +328,7 @@
 #if defined(__OpenBSD__) || defined(__FreeBSD__)
     static const char *const sysLExtDir = "/usr/local/share/mozilla/extensions";
 #else
-    static const char *const sysLExtDir = "/usr/share/mozilla/extensions";
+    static const char *const sysLExtDir = "/usr/share/icedove/extensions";
 #endif
     return NS_NewNativeLocalFile(nsDependentCString(sysLExtDir),
                                  false, aFile);
@@ -1129,11 +1129,11 @@
 #elif defined(XP_UNIX)
   static const char *const sysSExtDir = 
 #ifdef HAVE_USR_LIB64_DIR
-    "/usr/lib64/mozilla/extensions";
+    "/usr/lib64/icedove/extensions";
 #elif defined(__OpenBSD__) || defined(__FreeBSD__)
     "/usr/local/lib/mozilla/extensions";
 #else
-    "/usr/lib/mozilla/extensions";
+    "/usr/lib/icedove/extensions";
 #endif
 
   rv = NS_NewNativeLocalFile(nsDependentCString(sysSExtDir), false,
