--- icedove-3.0.old/comm-1.9.1/mozilla/configure.orig	2009-12-18 15:06:50.144206912 +0100
+++ icedove-3.0.old/comm-1.9.1/mozilla/configure	2009-12-18 15:07:35.791285838 +0100
@@ -13390,8 +13390,8 @@
   ;;
 
 standalone) 
-  MOZ_APP_NAME=mozilla
-  MOZ_APP_DISPLAYNAME=Mozilla
+  MOZ_APP_NAME=icedove
+  MOZ_APP_DISPLAYNAME=Icedove
   MOZ_APP_VERSION=$MOZILLA_VERSION
   ;;
 
--- icedove-3.0.old/comm-1.9.1/mozilla/configure.in.orig	2009-12-18 15:07:49.984408790 +0100
+++ icedove-3.0.old/comm-1.9.1/mozilla/configure.in	2009-12-18 15:08:12.906912163 +0100
@@ -4664,8 +4664,8 @@
   ;;
 
 standalone) 
-  MOZ_APP_NAME=mozilla
-  MOZ_APP_DISPLAYNAME=Mozilla
+  MOZ_APP_NAME=icedove
+  MOZ_APP_DISPLAYNAME=Icedove
   MOZ_APP_VERSION=$MOZILLA_VERSION
   ;;
 
--- icedove-3.0.old/comm-1.9.1/mail/confvars.sh	2009-12-05 02:53:48.000000000 +0100
+++ icedove-3.0/comm-1.9.1.new/mail/confvars.sh	2009-12-21 09:55:06.346526172 +0100
@@ -36,7 +36,7 @@
 #
 # ***** END LICENSE BLOCK *****
 
-MOZ_APP_NAME=thunderbird
+MOZ_APP_NAME=icedove
 MOZ_UPDATER=1
 MOZ_THUNDERBIRD=1
 MOZ_NO_ACTIVEX_SUPPORT=1
diff -ru icedove-3.0.old/comm-1.9.1/mail/branding/nightly/branding.nsi icedove-3.0/comm-1.9.1/mail/branding/nightly/branding.nsi
--- icedove-3.0.old/comm-1.9.1/mail/branding/nightly/branding.nsi	2010-01-13 12:50:52.355011705 +0100
+++ icedove-3.0/comm-1.9.1/mail/branding/nightly/branding.nsi	2010-01-13 12:52:49.625208158 +0100
@@ -39,7 +39,7 @@
 
 # BrandFullNameInternal is used for some registry and file system values
 # instead of BrandFullName and typically should not be modified.
-!define BrandFullNameInternal "Shredder"
+!define BrandFullNameInternal "Icedove"
 !define CompanyName           "mozilla.org"
 !define URLInfoAbout          "http://www.mozilla.org/"
 !define URLUpdateInfo         "http://www.mozilla.org/products/thunderbird/"
diff -ru icedove-3.0.old/comm-1.9.1/mail/branding/nightly/configure.sh icedove-3.0/comm-1.9.1/mail/branding/nightly/configure.sh
--- icedove-3.0.old/comm-1.9.1/mail/branding/nightly/configure.sh	2010-01-13 12:50:52.355011705 +0100
+++ icedove-3.0/comm-1.9.1/mail/branding/nightly/configure.sh	2010-01-13 12:52:49.625208158 +0100
@@ -1 +1 @@
-MOZ_APP_DISPLAYNAME=Shredder
+MOZ_APP_DISPLAYNAME=Icedove
diff -ru icedove-3.0.old/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.dtd icedove-3.0/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.dtd
--- icedove-3.0.old/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.dtd	2010-01-13 12:50:52.358344857 +0100
+++ icedove-3.0/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.dtd	2010-01-13 12:53:00.268553944 +0100
@@ -1,3 +1,3 @@
-<!ENTITY  brandShortName        "Shredder">
-<!ENTITY  brandFullName         "Shredder">
+<!ENTITY  brandShortName        "Icedove">
+<!ENTITY  brandFullName         "Icedove">
 <!ENTITY  vendorShortName       "mozilla.org">
diff -ru icedove-3.0.old/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.properties icedove-3.0/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.properties
--- icedove-3.0.old/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.properties	2010-01-13 12:50:52.358344857 +0100
+++ icedove-3.0/comm-1.9.1/mail/branding/nightly/locales/en-US/brand.properties	2010-01-13 12:53:00.268553944 +0100
@@ -1,4 +1,4 @@
-brandShortName=Shredder
-brandFullName=Shredder
+brandShortName=Icedove
+brandFullName=Icedove
 vendorShortName=mozilla.org
 
diff -ru icedove-3.0.old/comm-1.9.1/mail/installer/windows/nsis/shared.nsh icedove-3.0/comm-1.9.1/mail/installer/windows/nsis/shared.nsh
--- icedove-3.0.old/comm-1.9.1/mail/installer/windows/nsis/shared.nsh	2010-01-13 12:50:52.375011483 +0100
+++ icedove-3.0/comm-1.9.1/mail/installer/windows/nsis/shared.nsh	2010-01-13 12:53:13.349789738 +0100
@@ -518,19 +518,19 @@
     DeleteRegKey HKCU "$0\.eml\DefaultIcon"
   ${EndUnless}
 
-  ; Remove the Shredder clients key if its default icon contains thunderbird.exe
+  ; Remove the Icedove clients key if its default icon contains thunderbird.exe
   ClearErrors
-  ReadRegStr $1 HKLM "SOFTWARE\clients\mail\Shredder\DefaultIcon" ""
+  ReadRegStr $1 HKLM "SOFTWARE\clients\mail\Icedove\DefaultIcon" ""
   ${WordFind} "$1" "${FileMainEXE}" "E+1{" $R1
   ${Unless} ${Errors}
-    DeleteRegKey HKLM "SOFTWARE\clients\mail\Shredder"
+    DeleteRegKey HKLM "SOFTWARE\clients\mail\Icedove"
   ${EndUnless}
 
   ClearErrors
-  ReadRegStr $1 HKLM "SOFTWARE\clients\news\Shredder\DefaultIcon" ""
+  ReadRegStr $1 HKLM "SOFTWARE\clients\news\Icedove\DefaultIcon" ""
   ${WordFind} "$1" "${FileMainEXE}" "E+1{" $R1
   ${Unless} ${Errors}
-    DeleteRegKey HKLM "SOFTWARE\clients\news\Shredder"
+    DeleteRegKey HKLM "SOFTWARE\clients\news\Icedove"
   ${EndUnless}
 
   ; The Vista shim for 1.5.0.10 writes out a set of bogus keys which we need to
