--- comm-1.9.1/mozilla/configure.orig	2009-12-18 15:06:50.144206912 +0100
+++ comm-1.9.1/mozilla/configure	2009-12-18 15:07:35.791285838 +0100
@@ -13390,8 +13390,8 @@
   ;;
 
 standalone) 
-  MOZ_APP_NAME=mozilla
-  MOZ_APP_DISPLAYNAME=Mozilla
+  MOZ_APP_NAME=icedove
+  MOZ_APP_DISPLAYNAME=Icedove
   MOZ_APP_VERSION=$MOZILLA_VERSION
   ;;
 
--- comm-1.9.1/mozilla/configure.in.orig	2009-12-18 15:07:49.984408790 +0100
+++ comm-1.9.1/mozilla/configure.in	2009-12-18 15:08:12.906912163 +0100
@@ -4664,8 +4664,8 @@
   ;;
 
 standalone) 
-  MOZ_APP_NAME=mozilla
-  MOZ_APP_DISPLAYNAME=Mozilla
+  MOZ_APP_NAME=icedove
+  MOZ_APP_DISPLAYNAME=Icedove
   MOZ_APP_VERSION=$MOZILLA_VERSION
   ;;
 
