diff -burNp mozilla/configure mozilla-dud/configure
--- mozilla/configure	2008-10-19 18:21:23.000000000 +0200
+++ mozilla-dud/configure	2009-03-19 07:43:20.515129534 +0100
@@ -13073,8 +13073,8 @@ fi
 
 case "$MOZ_BUILD_APP" in
 suite)
-  MOZ_APP_NAME=seamonkey
-  MOZ_APP_DISPLAYNAME=SeaMonkey
+  MOZ_APP_NAME=icedove
+  MOZ_APP_DISPLAYNAME=Icedove
   MOZ_MAIL_NEWS=1
   MOZ_LDAP_XPCOM=1
   MOZ_COMPOSER=1
diff -burNp mozilla/configure.in mozilla-dud/configure.in
--- mozilla/configure.in	2008-10-19 18:14:06.000000000 +0200
+++ mozilla-dud/configure.in	2009-03-19 07:43:47.127624751 +0100
@@ -4305,8 +4305,8 @@ MOZ_ARG_ENABLE_STRING(application,
 
 case "$MOZ_BUILD_APP" in
 suite)
-  MOZ_APP_NAME=seamonkey
-  MOZ_APP_DISPLAYNAME=SeaMonkey
+  MOZ_APP_NAME=icedove
+  MOZ_APP_DISPLAYNAME=Icedove
   MOZ_MAIL_NEWS=1
   MOZ_LDAP_XPCOM=1
   MOZ_COMPOSER=1
diff -burNp mozilla/mail/app/nsMailApp.cpp mozilla-dud/mail/app/nsMailApp.cpp
--- mozilla/mail/app/nsMailApp.cpp	2009-01-01 04:44:17.000000000 +0100
+++ mozilla-dud/mail/app/nsMailApp.cpp	2009-03-19 07:44:29.744289595 +0100
@@ -48,7 +48,7 @@ static const nsXREAppData kAppData = {
   sizeof(nsXREAppData),
   nsnull,
   nsnull,
-  "Thunderbird",
+  "Icedove",
   NS_STRINGIFY(APP_VERSION),
   NS_STRINGIFY(BUILD_ID),
   "{3550f703-e582-4d05-9a08-453d09bdfdc6}",
diff -burNp mozilla/mail/app/profile/all-thunderbird.js mozilla-dud/mail/app/profile/all-thunderbird.js
--- mozilla/mail/app/profile/all-thunderbird.js	2008-06-03 11:31:22.000000000 +0200
+++ mozilla-dud/mail/app/profile/all-thunderbird.js	2009-03-19 07:48:38.883639809 +0100
@@ -35,7 +35,7 @@
  *
  * ***** END LICENSE BLOCK ***** */
 
-#expand pref("general.useragent.extra.thunderbird", "Thunderbird/__APP_VERSION__");
+#expand pref("general.useragent.extra.thunderbird", "Icedove/__APP_VERSION__");
 
 #expand pref("general.useragent.locale", "__AB_CD__");
 pref("general.skins.selectedSkin", "classic/1.0");
@@ -74,10 +74,10 @@ pref("app.update.vendorName.override", "
 
 // URL user can browse to manually if for some reason all update installation
 // attempts fail.  TODO: Change this URL
-pref("app.update.url.manual", "http://%LOCALE%.www.mozilla.com/%LOCALE%/%APP%/");
+pref("app.update.url.manual", "http://%LOCALE%.www.mozilla.com/%LOCALE%/thunderbird/");
 // A default value for the "More information about this update" link
 // supplied in the "An update is available" page of the update wizard. 
-pref("app.update.url.details", "http://%LOCALE%.www.mozilla.com/%LOCALE%/%APP%/releases/");
+pref("app.update.url.details", "http://%LOCALE%.www.mozilla.com/%LOCALE%/thunderbird/releases/");
 // User-settable override to app.update.url for testing purposes.
 //pref("app.update.url.override", "");
 
@@ -102,7 +102,7 @@ pref("app.update.timer", 600000);
 pref("app.update.showInstalledUI", false);
 
 // Release notes URL
-pref("app.releaseNotesURL", "http://%LOCALE%.www.mozilla.com/%LOCALE%/%APP%/%VERSION%/releasenotes/");
+pref("app.releaseNotesURL", "http://%LOCALE%.www.mozilla.com/%LOCALE%/thunderbird/%VERSION%/releasenotes/");
 
 // Blocklist preferences
 pref("extensions.blocklist.enabled", true);
@@ -130,8 +130,8 @@ pref("extensions.update.url", "chrome://
 pref("extensions.update.interval", 86400);  // Check for updates to Extensions and 
                                             // Themes every week
 // Non-symmetric (not shared by extensions) extension-specific [update] preferences
-pref("extensions.getMoreExtensionsURL", "https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/extensions/");
-pref("extensions.getMoreThemesURL", "https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/themes/");
+pref("extensions.getMoreExtensionsURL", "https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/thunderbird/%VERSION%/extensions/");
+pref("extensions.getMoreThemesURL", "https://%LOCALE%.add-ons.mozilla.com/%LOCALE%/thunderbird/%VERSION%/themes/");
 pref("extensions.dss.enabled", false);          // Dynamic Skin Switching                                               
 pref("extensions.dss.switchPending", false);    // Non-dynamic switch pending after next
 
diff -burNp mozilla/mail/app/profile/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}/install.rdf.in mozilla-dud/mail/app/profile/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}/install.rdf.in
--- mozilla/mail/app/profile/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}/install.rdf.in	2005-08-17 19:17:15.000000000 +0200
+++ mozilla-dud/mail/app/profile/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}/install.rdf.in	2009-03-19 07:49:56.700298615 +0100
@@ -20,7 +20,7 @@
     </em:targetApplication>
     
     <!-- Front End MetaData -->
-    <em:name>Thunderbird (default)</em:name>
+    <em:name>Icedove (default)</em:name>
     <em:description>The default theme</em:description>
 
     <!-- EXTENSION AUTHORS!
diff -burNp mozilla/mail/base/content/credits.xhtml mozilla-dud/mail/base/content/credits.xhtml
--- mozilla/mail/base/content/credits.xhtml	2009-01-01 04:44:18.000000000 +0100
+++ mozilla-dud/mail/base/content/credits.xhtml	2009-03-19 07:51:37.976955410 +0100
@@ -216,7 +216,7 @@
     </div>
 
     <div id="creditsBox">
-        <h2 class="title">&brandFullName;&trade;
+        <h2 class="title">&brandFullName;
           <div class="motto">&brandMotto;</div>
         </h2>
 
@@ -348,19 +348,13 @@
         <p id="gecko" class="center">&credit.poweredByGecko;</p>
 
         <p class="footnote">
-          &brandFullName;&trade; &license.part0; &copy;1998-2009 &license.part1;
+          &brandFullName; &license.part0; &copy;1998-2009 &license.part1;
           <a href="" link="http://www.mozilla.org/credits/" onclick="visitLink(event); event.preventDefault();">&license.contrib;</a>,
           &license.part2;
           <a href="" onclick="openLicense(); event.preventDefault();">license</a>
           &license.part3;</p>
 
         <p class="footnote">
-          Mozilla Thunderbird&trade; and the Thunderbird logo are trademarks of the Mozilla
-          Foundation. You are not granted rights or licenses to the trademarks
-          of the Mozilla Foundation or any party, including without limitation the
-          Thunderbird name or logo.</p>
-
-        <p class="footnote">
           U.S. GOVERNMENT END USERS. The Software is a &ldquo;commercial item,&rdquo;
           as that term is defined in 48 C.F.R. 2.101 (Oct. 1995), consisting of
           &ldquo;commercial computer software&rdquo; and &ldquo;commercial computer software
diff -burNp mozilla/mail/base/content/mailWindowOverlay.xul mozilla-dud/mail/base/content/mailWindowOverlay.xul
--- mozilla/mail/base/content/mailWindowOverlay.xul	2008-05-10 08:36:09.000000000 +0200
+++ mozilla-dud/mail/base/content/mailWindowOverlay.xul	2009-03-19 07:54:01.656941108 +0100
@@ -1893,17 +1893,10 @@
      <menupopup id="menu_HelpPopup" onpopupshowing="buildHelpMenu();">
 #ifdef XP_MACOSX
           <menuitem id="hintsAndTips" label="&hintsAndTips.label;" key="key_macHelp" oncommand="openRegionURL('mailnews.hints_and_tips.url');"/>
-          <menuitem id="releaseNotes" accesskey="&releaseCmd.accesskey;" label="&releaseCmd.label;" oncommand="openFormattedRegionURL('app.releaseNotesURL');"/>
 #else
-          <menuitem id="releaseNotes" accesskey="&releaseCmd.accesskey;" label="&releaseCmd.label;" oncommand="openFormattedRegionURL('app.releaseNotesURL');"/>
           <menuitem id="hintsAndTips" accesskey="&hintsAndTips.accesskey;" label="&hintsAndTips.label;" oncommand="openRegionURL('mailnews.hints_and_tips.url');"/>
 #endif
           <menuseparator id="menu_HelpAboutSeparator"/>
-          <menuitem id="checkForUpdates" accesskey="&updateCmd.accesskey;" class="menuitem-iconic"
-                    label="&updateCmd.label;" oncommand="checkForUpdates();"/>
-#ifndef XP_MACOSX
-          <menuseparator id="menu_HelpAfterUpdatesSeparator"/>
-#endif
           <menuitem id="aboutName" accesskey="&aboutCmd.accesskey;"
                     label="&aboutCmd.label;"
 #ifdef XP_MACOSX
diff -burNp mozilla/mail/base/content/start.xhtml mozilla-dud/mail/base/content/start.xhtml
--- mozilla/mail/base/content/start.xhtml	2007-02-02 01:15:54.000000000 +0100
+++ mozilla-dud/mail/base/content/start.xhtml	2009-03-19 07:54:48.966936095 +0100
@@ -145,7 +145,6 @@ a:active, a:hover {
 
 <h2>&startpage.moreInfo;</h2>
 <p>&startpage.helpInfo; <a href="&startpage.helpURL;">&startpage.help;</a>.</p>
-<p>&startpage.pageInfo; <a href="&startpage.pageURL;">&startpage.page;</a>.</p>
 
 <!-- Localization-specific information, i.e. how to download a dictionary. -->
 &startpage.extra;
diff -burNp mozilla/mail/components/compose/content/messengercompose.xul mozilla-dud/mail/components/compose/content/messengercompose.xul
--- mozilla/mail/components/compose/content/messengercompose.xul	2007-05-01 02:40:55.000000000 +0200
+++ mozilla-dud/mail/components/compose/content/messengercompose.xul	2009-03-19 07:55:35.016931436 +0100
@@ -542,9 +542,7 @@
         <menupopup id="menu_HelpPopup">
 #ifdef XP_MACOSX
           <menuitem id="hintsAndTips" label="&hintsAndTips.label;" key="key_macHelp" oncommand="openRegionURL('mailnews.hints_and_tips.url');"/>
-          <menuitem id="releaseNotes" accesskey="&releaseCmd.accesskey;" label="&releaseCmd.label;" oncommand="openFormattedRegionURL('app.releaseNotesURL');"/>
 #else
-          <menuitem id="releaseNotes" accesskey="&releaseCmd.accesskey;" label="&releaseCmd.label;" oncommand="openFormattedRegionURL('app.releaseNotesURL');"/>
           <menuitem id="hintsAndTips" accesskey="&hintsAndTips.accesskey;" label="&hintsAndTips.label;" oncommand="openRegionURL('mailnews.hints_and_tips.url');"/>
           <menuseparator id="menu_HelpAboutSeparator"/>
 #endif
diff -burNp mozilla/mail/components/preferences/advanced.xul mozilla-dud/mail/components/preferences/advanced.xul
--- mozilla/mail/components/preferences/advanced.xul	2006-12-04 06:43:32.000000000 +0100
+++ mozilla-dud/mail/components/preferences/advanced.xul	2009-03-19 07:56:17.306927413 +0100
@@ -200,11 +200,6 @@
         <tabpanel orient="vertical" align="start">
           <label>&autoCheck.label;</label>
           <vbox class="indent">
-            <checkbox id="enableAppUpdate"
-                      label="&enableAppUpdate.label;"
-                      accesskey="&enableAppUpdate.accesskey;"
-                      preference="app.update.enabled"
-                      xhtml2:role="wairole:checkbox" aaa:describedby="updateInfo"/>
             <checkbox id="enableAddonUpdate"
                       label="&enableAddonsUpdate.label;"
                       accesskey="&enableAddonsUpdate.accesskey;"
diff -burNp mozilla/mail/locales/en-US/chrome/messenger/aboutDialog.dtd mozilla-dud/mail/locales/en-US/chrome/messenger/aboutDialog.dtd
--- mozilla/mail/locales/en-US/chrome/messenger/aboutDialog.dtd	2009-01-01 04:44:18.000000000 +0100
+++ mozilla-dud/mail/locales/en-US/chrome/messenger/aboutDialog.dtd	2009-03-19 07:57:48.853584778 +0100
@@ -2,6 +2,4 @@
 <!ENTITY credits                "Credits">
 <!ENTITY aboutLink              "&lt; About &brandFullName;">
 <!ENTITY aboutVersion           "version">
-<!ENTITY copyrightText          "&#169;1998-2009 Contributors. All Rights Reserved. Mozilla Thunderbird and the 
-                                 Thunderbird logos are trademarks of the Mozilla Foundation.  All rights 
-                                 reserved.">
+<!ENTITY copyrightText          "&#169;1998-2009 Contributors. All Rights Reserved.">
diff -burNp mozilla/mail/locales/en-US/chrome/messenger/credits.dtd mozilla-dud/mail/locales/en-US/chrome/messenger/credits.dtd
--- mozilla/mail/locales/en-US/chrome/messenger/credits.dtd	2005-06-25 00:43:00.000000000 +0200
+++ mozilla-dud/mail/locales/en-US/chrome/messenger/credits.dtd	2009-03-19 07:58:30.440247407 +0100
@@ -1,5 +1,5 @@
 <!ENTITY credit.title     "&brandFullName; Credits">
-<!ENTITY brandMotto       "Thunderbirds Are Go!">
+<!ENTITY brandMotto       "Icedovers Are Free!">
 <!ENTITY credit.leads     "Engineering Leads">
 <!ENTITY credit.core      "Core Development Posse">
 <!ENTITY credit.gecko     "Gecko Layout Engine">
diff -burNp mozilla/mail/locales/en-US/chrome/messenger/start.dtd mozilla-dud/mail/locales/en-US/chrome/messenger/start.dtd
--- mozilla/mail/locales/en-US/chrome/messenger/start.dtd	2005-07-27 22:20:36.000000000 +0200
+++ mozilla-dud/mail/locales/en-US/chrome/messenger/start.dtd	2009-03-19 07:59:21.473575429 +0100
@@ -19,11 +19,11 @@
 
 <!ENTITY startpage.moreInfo "More Information">
 
-<!ENTITY startpage.help "Thunderbird Help">
+<!ENTITY startpage.help "Icedove Help">
 <!ENTITY startpage.helpInfo "For frequently asked questions, tips and general help, visit">
 <!ENTITY startpage.helpURL "http://www.mozilla.org/support/thunderbird/">
 
-<!ENTITY startpage.page "Thunderbird Product Page">
+<!ENTITY startpage.page "Icedove Product Page">
 <!ENTITY startpage.pageInfo "For product information, visit the">
 <!ENTITY startpage.pageURL "http://www.mozilla.org/products/thunderbird/">
 
diff -burNp mozilla/mail/locales/en-US/chrome/messenger-newsblog/newsblog.properties mozilla-dud/mail/locales/en-US/chrome/messenger-newsblog/newsblog.properties
--- mozilla/mail/locales/en-US/chrome/messenger-newsblog/newsblog.properties	2005-08-25 21:31:58.000000000 +0200
+++ mozilla-dud/mail/locales/en-US/chrome/messenger-newsblog/newsblog.properties	2009-03-19 08:00:15.153570170 +0100
@@ -9,8 +9,8 @@ subscribe-errorOpeningFile=Could not ope
 
 subscribe-OPMLImportTitle=Select OPML file to import
 subscribe-OPMLExportTitle=Export feeds as an OPML file
-subscribe-OPMLExportFileTitle=Thunderbird OPML Export
-subscribe-OPMLExportFileName=MyThunderbirdFeeds.opml
+subscribe-OPMLExportFileTitle=Icedove OPML Export
+subscribe-OPMLExportFileName=MyIcedoveFeeds.opml
 #LOCALIZATION NOTE: %S is the name of the OPML file the user tried to import.
 subscribe-errorInvalidOPMLFile=Either %S is not a valid OPML file or there was an error importing the file.
 
diff -burNp mozilla/mail/locales/en-US/installer/installer.inc mozilla-dud/mail/locales/en-US/installer/installer.inc
--- mozilla/mail/locales/en-US/installer/installer.inc	2006-02-28 22:09:08.000000000 +0100
+++ mozilla-dud/mail/locales/en-US/installer/installer.inc	2009-03-19 08:00:59.893566018 +0100
@@ -19,7 +19,7 @@ Type_SubTitle=Choose setup options.
 Type_Message0=Choose the type of Setup you prefer, then click Next.
 
 Standard_Short=&Standard
-Standard_Long=Thunderbird will be installed with the most common options.
+Standard_Long=Icedove will be installed with the most common options.
 
 Custom_Short=&Custom
 Custom_Long=You may choose individual options to be installed. Recommended for experienced users.
diff -burNp mozilla/mail/locales/en-US/updater/updater.ini mozilla-dud/mail/locales/en-US/updater/updater.ini
--- mozilla/mail/locales/en-US/updater/updater.ini	2006-09-25 20:36:18.000000000 +0200
+++ mozilla-dud/mail/locales/en-US/updater/updater.ini	2009-03-19 08:01:25.760229881 +0100
@@ -1,4 +1,4 @@
 ; This file is in the UTF-8 encoding
 [Strings]
 Title=Software Update
-Info=Thunderbird is installing your updates and will start in a few moments...
+Info=Icedove is installing your updates and will start in a few moments...
diff -burNp mozilla/toolkit/locales/en-US/chrome/mozapps/extensions/extensions.properties mozilla-dud/toolkit/locales/en-US/chrome/mozapps/extensions/extensions.properties
--- mozilla/toolkit/locales/en-US/chrome/mozapps/extensions/extensions.properties	2006-09-11 22:38:12.000000000 +0200
+++ mozilla-dud/toolkit/locales/en-US/chrome/mozapps/extensions/extensions.properties	2009-03-19 08:03:14.483552087 +0100
@@ -32,8 +32,8 @@ disableWarningDependMessage=If you disab
 disableQueryMessage=Do you want to disable %S?
 
 extensions.update.url=https://addons.mozilla.org/update/VersionCheck.php?reqVersion=%REQ_VERSION%&id=%ITEM_ID%&version=%ITEM_VERSION%&maxAppVersion=%ITEM_MAXAPPVERSION%&status=%ITEM_STATUS%&appID=%APP_ID%&appVersion=%APP_VERSION%&appOS=%APP_OS%&appABI=%APP_ABI%
-extensions.getMoreExtensionsURL=http://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/extensions/
-extensions.getMoreThemesURL=http://%LOCALE%.add-ons.mozilla.com/%LOCALE%/%APP%/%VERSION%/themes/
+extensions.getMoreExtensionsURL=http://%LOCALE%.add-ons.mozilla.com/%LOCALE%/thunderbird/%VERSION%/extensions/
+extensions.getMoreThemesURL=http://%LOCALE%.add-ons.mozilla.com/%LOCALE%/thunderbird/%VERSION%/themes/
 
 themesTitle=Themes
 extensionsTitle=Extensions
diff -burNp mozilla/toolkit/xre/nsAppRunner.cpp mozilla-dud/toolkit/xre/nsAppRunner.cpp
--- mozilla/toolkit/xre/nsAppRunner.cpp	2007-11-08 12:56:25.000000000 +0100
+++ mozilla-dud/toolkit/xre/nsAppRunner.cpp	2009-03-19 08:05:20.350206255 +0100
@@ -1121,7 +1121,7 @@ VerifyInstallation(nsIFile* aAppDir)
 static void
 DumpVersion()
 {
-  printf("%s %s %s, %s\n", gAppData->vendor ? gAppData->vendor : "", gAppData->name, gAppData->version, gAppData->copyright);
+  printf("%s %s, %s\n", gAppData->name, gAppData->version, gAppData->copyright);
 }
 
 #ifdef MOZ_ENABLE_XREMOTE
