--- mozilla.orig/mail/app/profile/all-thunderbird.js	2008-06-23 14:04:11.000000000 +0200
+++ mozilla/mail/app/profile/all-thunderbird.js	2008-06-24 13:22:48.000000000 +0200
@@ -78,7 +78,7 @@
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make     
 // the UI easier to construct.
-pref("app.update.auto", true);
+pref("app.update.auto", false);
 
 // Defines how the Application Update Service notifies the user about updates:
 //
@@ -373,7 +373,7 @@
 pref("alerts.totalOpenTime", 3000);
 
 // analyze urls in mail messages for scams
-pref("mail.phishing.detection.enabled", true);
+pref("mail.phishing.detection.enabled", false);
 // If phishing detection is enabled, allow fine grained control
 // of the local, static tests
 pref("mail.phishing.detection.ipaddresses", true); 
--- mozilla.orig/mozilla/modules/libpref/init/all.js
+++ mozilla/mozilla/modules/libpref/init/all.js
@@ -1053,6 +1053,7 @@ pref("javascript.options.ion.offthread_compilation", true);
 // memory, but makes things like Function.prototype.toSource()
 // fail.
 pref("javascript.options.discardSystemSource", false);
+pref("javascript.options.showInConsole",    true);
 // This preference limits the memory usage of javascript.
 // If you want to change these values for your device,
 // please find Bug 417052 comment 17 and Bug 456721
