--- mozilla/mozilla/toolkit/system/gnome/nsAlertsIconListener.cpp~	2011-03-03 12:42:51.000000000 +0100
+++ mozilla/mozilla/toolkit/system/gnome/nsAlertsIconListener.cpp	2011-04-06 18:10:55.850586553 +0200
@@ -204,7 +204,11 @@
 {
   NotifyNotification* notify = notify_notification_new(mAlertTitle.get(),
                                                        mAlertText.get(),
-                                                       NULL, NULL);
+                                                       NULL
+#if !defined(NOTIFY_VERSION_MINOR) || (NOTIFY_VERSION_MAJOR == 0 && NOTIFY_VERSION_MINOR < 7)
+						       , NULL
+#endif
+						       );
   if (!notify)
     return NS_ERROR_OUT_OF_MEMORY;
 
