--- mozilla/configure.in~	2010-03-17 19:02:26.000000000 +0200
+++ mozilla/configure.in	2010-04-09 14:19:51.573033459 +0300
@@ -4953,6 +4953,13 @@
     MOZ_LDAP_XPCOM=,
     MOZ_LDAP_XPCOM=1)
 
+dnl system LDAP Support
+dnl ========================================================
+MOZ_ARG_WITH_STRING(system-ldap,
+[  --with-system-ldap[=PFX]
+                          Use system mozldap [installed at prefix PFX]],
+    MOZ_LDAP_DIR=$withval)
+
 dnl ========================================================
 dnl = Trademarked Branding 
 dnl ========================================================
@@ -7625,7 +7632,7 @@
 # if we're building the LDAP XPCOM component, we need to build 
 # the c-sdk first.  
 #
-if test "$MOZ_LDAP_XPCOM"; then
+if test "$MOZ_LDAP_XPCOM" && test -z "$MOZ_LDAP_DIR"; then
 
     # these subdirs may not yet have been created in the build tree.
     # don't use the "-p" switch to mkdir, since not all platforms have it
